docs/user_guide/upgrading.rst

```diff
@@ -0,0 +1,38 @@
+Upgrading
+==========
+See also [CHANGES.md](https://github.com/fairlearn/fairlearn/blob/master/CHANGES.md).
```

Comment:
This should probably be in ReST, i.e.
```
`CHANGES.md <https://github.com/fairlearn/fairlearn/blob/master/CHANGES.md>`_
```

Reply:
Two other ReST style things that I noticed in this file:

- header underline needs to be at least as long as the header text
  ```
  Upgrading from 0.2 to 0.3
  -------------------------
  ```
- empty line after header underline (not 100% sure if this is strictly required)

Reply:
thanks! 👍